integrations/Hotjar/browser.js

```diff
@@ -41,18 +45,30 @@ class Hotjar {
   }
 
   track(rudderElement) {
-    logger.debug("[Hotjar] track:: method not supported");
+    logger.debug("===In Hotjar track===");
+
+    const { event } = rudderElement.message;
+
+    if (!event) {
```

Comment:
We should also clip the message to 750 characters and replace whitespace with `_`.

Reply:
resolved